xmipp

```diff
@@ -47,20 +47,22 @@ SCIPION_EM_XMIPP = 'scipion-em-xmipp'
 CUFFTADVISOR = 'cuFFTAdvisor'
 CTPL = 'CTPL'
 GTEST = 'googletest'
+LIBSVM = 'libsvm'
 
 REPOSITORIES = {XMIPP: 'https://github.com/I2PC/xmipp.git',
                 XMIPP_CORE: 'https://github.com/I2PC/xmippCore.git',
                 XMIPP_VIZ: 'https://github.com/I2PC/xmippViz.git',
                 SCIPION_EM_XMIPP: 'https://github.com/I2PC/scipion-em-xmipp.git',
                 CUFFTADVISOR: 'https://github.com/DStrelak/cuFFTAdvisor.git',
                 CTPL: 'https://github.com/vit-vit/CTPL.git',
-                GTEST: 'https://github.com/google/googletest'}
+                GTEST: 'https://github.com/google/googletest',
+                LIBSVM: 'https://github.com/cossorzano/libsvm.git'}
```

Comment:
The next update of libSVM, we overwrite the fork and download the last release?

Reply:
I wonder if the fork should not better be done via the I2PC account

Reply:
Maybe, but I do not have the password for I2PC